dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -324,6 +327,15 @@ public void checkAndUpdateSubConfigs() {
         checkMock(interfaceClass);
     }
 
+    /**
+     * Determine if it is injvm
+     *
+     * @return
+     */
+    private boolean isJvm() {

```

Comment:
pls. change method name from **isJvm** to **isOnlyInJvm**

Reply:
get